src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -320,7 +319,7 @@ export default {
 
 			if (!this.model.attributes.audioAvailable) {
 				return {
-					content: t('spreed', 'No audio'),
+					content: t('spreed', 'No audio available. Select audio device'),
```

Comment:
Where? 😄 

I know where, but other users might not. Maybe clicking it could open the device picker? Or we at least tell the user where to click to open it?

Reply:
We could change it to "Click to select...", as this is a tooltip for described button